geodepy/heights_s3.py

```diff
@@ -0,0 +1,234 @@
+#___________________________________________________________________________#
+# Some notes:
+# Written by Jack McCubbine of Geoscience Australia, date: 08/11/2019
+# This code contains functions to handle tranformations between GPS and 
+# AWVS/AHD and Vice Versa 
+# Gridded data used for the varisous reference surfaces are geotiff files
+# These allow direct access remotely using "gdal"
+# File names are hard coded 
+# Some exmaples of how the functions can be used are give towards the end 
+#___________________________________________________________________________#
+# Import dependencies 
+import gdal
+import numpy as np
+from scipy.interpolate import griddata
+#___________________________________________________________________________#
+# Interpolation functions
+def interp_DOVPV(Lat,Long):
+    # Import the DOVPM file
+    f = gdal.Open('/vsicurl/https://geoid.s3-ap-southeast-2.amazonaws.com/AVWS/DOV_PV.tif')

```

Comment:
Is it possible to move the s3 file refs out from the function definitions into a set of variables at the start of the script? Something like geodepy.transform with the grs80 and utm variables (defined in the script rather than imported from elsewhere) and then added as default variables to the function calls?

i.e.
dovpv_s3 = '/vsicurl/https://geoid.s3-ap-southeast-2.amazonaws.com/AVWS/DOV_PV.tif'
...
def interp_DOVPV(Lat,Long,DOVPV=dovpv_s3)

This way, all the externally referenced files are easily viewable (and updateable!) at the top of the script.